scripts/hostapd/dnsmasq.conf

```diff
@@ -1,2 +1,2 @@
-dhcp-range=10.0.0.2,10.0.0.127,255.255.255.0,12h
+dhcp-range=10.0.0.2,10.0.0.126,12h
```

Comment:
Network has been changed to 10.0.0.1 so if you want to keep this it should be 
```dhcp-range=10.10.0.2,10.10.0.126,12h``` or the network reset to 10.0.0.1 below.

Reply:
My bad, these should be `10.10.0.x`